Cargo.toml

```diff
@@ -23,35 +23,27 @@ serde = "1.0.136"
 serde_derive = "1.0.103"
 serde_json = "1.0.79"
 serde_yaml = "0.8.23"
-mango = { path = "deps/mango-v3/program", default-features = false, features = ["no-entrypoint"] }
-mango-common = { path = "deps/mango-v3/common" }
-solana-bench-tps = { path = "deps/solana/bench-tps" }
-solana-clap-utils = { path = "deps/solana/clap-utils" }
-solana-cli-config = { path = "deps/solana/cli-config" }
-solana-client = { path = "deps/solana/client" }
-solana-core = { path = "deps/solana/core" }
-solana-faucet = { path = "deps/solana/faucet" }
-solana-genesis = { path = "deps/solana/genesis" }
-solana-gossip = { path = "deps/solana/gossip" }
-solana-logger = { path = "deps/solana/logger" }
-solana-measure = { path = "deps/solana/measure" }
-solana-metrics = { path = "deps/solana/metrics" }
-solana-net-utils = { path = "deps/solana/net-utils" }
-solana-rpc = { path = "deps/solana/rpc" }
-solana-runtime = { path = "deps/solana/runtime" }
-solana-sdk = { path = "deps/solana/sdk" }
-solana-streamer = { path = "deps/solana/streamer" }
-solana-transaction-status = { path = "deps/solana/transaction-status" }
-solana-version = { path = "deps/solana/version" }
+
+mango = { git = "https://github.com/blockworks-foundation/mango-v3.git", branch = "mango_bencher_compatible", default-features = false, features = ["no-entrypoint"] }
+mango-common = { git = "https://github.com/blockworks-foundation/mango-v3.git", branch = "mango_bencher_compatible" }
+
+solana-client = { git = "https://github.com/solana-labs/solana.git", branch="v1.14" }
```

Comment:
why not testmet version `1.15`?

Reply:
It break the TPU client interface. 
We first merge the other pull request to split the code in multiple files. then i move the code to v1.15